vispy/color/colormap.py

```diff
@@ -138,13 +141,13 @@ def _glsl_mix(controls=None, colors=None, texture_map_data=None):
         The first control point must be 0.0. The last control point must be
         1.0. The number of control points depends on the interpolation scheme.
 
-    texture_map_data : Numpy array of size of 1D texture lookup data for 
-                       luminance to RGBA conversion.
-                       The size of np.float32 array is (LUT_len, 4).
-                       If texture_map_data is not None,
-                       the RGBA data for the array is computed.
+    texture_map_data : Numpy array of size of 1D texture lookup data for
```

Comment:
`: ndarray, shape (n_colors,)`

Also the `controls` should just be `controls : list` and the rest should be description, following more or less [the numpy docstring standard](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)